Examples/OVHAPIWrapper-Example-watchOS/OVHAPIWrapper-Example-watchOS WatchKit Extension/VPSRowController.swift

```diff
@@ -46,7 +46,7 @@ class VPSRowController: NSObject {
                 
                 var color = UIColor.grayColor()
                 if vps.state == .running {
-                    color = UIColor.greenColor()
+                    color = UIColor(red: 0.0, green: 0.8, blue: 0.2, alpha: 1.0)
```

Comment:
It is in the example project, not in the warpper framework, do we really need this?